src/services/API/RootClient.js

```diff
@@ -28,6 +29,13 @@ client.interceptors.request.use(
       if (store.state.auth.accessToken) {
         config.headers.Authorization = `Bearer ${store.state.auth.accessToken.access_token}`;
       }
+
+      let currentRoute = Router.currentRoute.value;
+      if (currentRoute.params.org != undefined) {
+        store.dispatch("auth/setActiveWorkspace", currentRoute.params.org);
```

Comment:
if someone doesn't belong to an organization and if they are visiting `/af/play/PLIO_ID` this will set their workspace to AF right? Even when they don't belong to that workspace?

Reply:
Yes, this will set it to that.

We will have the backend address the API permissioning part and return corresponding Response status.